### DISCUSSION TITLE
Pi Camera Solar Panel BOM

### DISCUSSION THREAD

**Discussion by rgrokett** [alanbjohnston/CubeSatSim#235] [Ideas]
I noticed that the [6.1 Solar Panel assembly](https://github.com/alanbjohnston/CubeSatSim/wiki/6.-Solar-Panel-and-Frame) for use with Pi Camera shows 90mm x 35mm solar cells while the BOM spreadsheet shows  Solar Cells 57mm x 28mm. 

I realize that other than "looks" this doesn't have significant impact as the smaller ones can still work, but just wanted to let folks know.

**Comment by alanbjohnston** [comment]
Thanks for the comment. Yes, the 80mm x 35mm panels don't seem to be available anymore - that's why the BOM has been updated.  I have updated the Wiki instructions to indicate this.

Alan
